custom_model_runner/datarobot_drum/resource/transform_helpers.py

```diff
@@ -66,3 +67,16 @@ def validate_transformed_output(transformed_output, should_be_sparse=False):
     else:
         assert type(transformed_output) == pd.DataFrame
         assert transformed_output.shape[1] == 10
+
+
+def parse_multi_part_response(response):
```

Comment:
@4dahalibut  this is a vaguely hacky way to pick apart a multi-part response and parse into a dict, but it does seem to work with all the data types we want to send and not require eval- line 175 of the predict_mixin above is how the response is now getting encoded.